.github/workflows/wheels-ci.yml

```diff
@@ -0,0 +1,43 @@
+name: Build NEURON Python wheels for CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build-test:
+    name: Build and test NEURON Python wheels
+    uses: ./.github/workflows/wheels-template.yml
+    with:
+      platform: ${{ matrix.os }}
+      python_version: ${{ matrix.python_version }}
+      commit: ${{ github.sha }}
```

Comment:
GitHub annoyingly sets `github.sha` to the _merge_ commit, so it cannot be checked out locally. The alternative is to use `github.event.pull_request.head.sha`, but then the CI on the main branch will get an empty input (I think it checks out the default, i.e. `master`, which _is_ the intended behavior, but I don't know if `required: true` accepts empty values as well).